src/util.h

```diff
@@ -69,15 +70,50 @@ inline std::string _(const char* psz)
 void SetupEnvironment();
 bool SetupNetworking();
 
+enum LogFlags : uint32_t {
+    LOG_NONE        = 0,
+    LOG_NET         = (1 <<  0),
```

Comment:
I could make it BCLOG or something like that, you're right about namespace collisions, my initial sed through to rename the LOG_ accidentally renamed some things it shouldn't.